### ISSUE TITLE
Fix 'seams' in some geometries in the GeoJSON

### ISSUE THREAD

**Issue by ryaanahmed** [dhmit/democracy_africa#33]
Follow on work from #31 -- see @JasonLin43212 's remarks there about a visible seam between Somaliland and Somalia. This probably needs manual intervention.

**Comment by mscuthbert** [comment]
We should consult @evanlieb on this, but my sense is that the US Government doesn't recognize Somaliland as separate from Somalia.  So for political reasons we should stick with keeping it in Somalia.

(Wikipedia says no other country recognizes its independence)

**Comment by evanlieb** [comment]
correct - no need to break out Somaliland in anything we do

***

Evan S Lieberman
Total Professor of Political Science and Contemporary Africa
Massachusetts Institute of Technology (MIT)
 
email: evanlieb@mit.edu
website: evanlieberman.org <https://evanlieberman.org/>

**Comment by ryaanahmed** [comment]
Yep, that's what we figured. The geometries we're using have it as a separate piece, so we've merged it with Somalia. This issue is just about removing the resulting 'seam' between the two pieces in the geometries.
